### PR TITLE
Fix: Convert to JSX (fixes #2 and #4)

### LINKED ISSUE #2: Allowfullscreen in iframe
It could be necessary/interesting to include inside the template to 'allowfullscreen'. If you invoke requestFullscreen event from an element, it is returning that it's not allowed. Similar to:

_<iframe src=" " width=" " height=" " frameborder="0" allowfullscreen webkitallowfullscreen mozallowfullscreen scrolling="no"></iframe>_

### LINKED ISSUE #4: Convert to JSX

### PR DESCRIPTION
**Pull request by zubairslamdien**
Fixes #2 and #4

### Fix

- Converts .hbs template to .jsx
- The view now inherits from ComponentView
- allowFullScreen attributes added to the iframe element

### CODE REVIEW

File: js/iframeView.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'core/js/views/componentView';
+
+class iframeView extends ComponentView {
```
Comment (oliverfoster) [inline js/iframeView.js]: Classes always have an uppercase name please.
```suggestion
class IFrameView extends ComponentView {
```

File: js/iframeView.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    postRender() {
+        let $iframe = this.$('iframe');
+
+        _.bindAll(this, 'onIframeLoaded', 'onMessage', 'onInview');
```
Comment (oliverfoster) [inline js/iframeView.js]: These go in `postInitialize`

**Comment by oliverfoster** [comment]
@zubairslamdien would you mind making the suggested changes?

**Review by oliverfoster — APPROVED** [review]

**Comment by oliverfoster** [comment]
Ready for review

**Comment by zubairslamdien** [comment]
Checked out and seems good to me. @oliverfoster

**Pull request merged**